lib/active_merchant/billing/gateway.rb

```diff
@@ -125,8 +125,9 @@ def generate_unique_id
       class_attribute :supported_cardtypes
       self.supported_cardtypes = []
 
-      class_attribute :currencies_without_fractions
+      class_attribute :currencies_without_fractions, :currencies_with_three_decimal_places
       self.currencies_without_fractions = %w(BIF BYR CLP CVE DJF GNF HUF ISK JPY KMF KRW PYG RWF UGX VND VUV XAF XOF XPF)
+      self.currencies_with_three_decimal_places = %w()
```

Comment:
I wonder if we should add them here as defaults. Seems we know which ones are and if it needs to change at the gateway level it can be overridden?